### DISCUSSION TITLE
Loki absent_over_time grouping

### DISCUSSION THREAD

**Discussion by suikast42** [grafana/grafana#66837] [Alerting]
I have defined a count_over_time query for my ingress stream  like this. 

![image](https://user-images.githubusercontent.com/5046525/233059752-2b78abee-3cfc-4a4a-a24e-842d3a4768cf.png)

But If I changed the query to absent_over_time then the grouping is go away. 

![image](https://user-images.githubusercontent.com/5046525/233060007-ce0070ce-288f-42af-9011-8639190bda8c.png)

Is it possible to group the result vector by label  with absent_over_time ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
